build.gradle.kts

```diff
@@ -105,3 +106,20 @@ tasks.wrapper {
         )
     }
 }
+
+idea {
+    module {
+        // Mark directories as excluded so that they don't appear in IntelliJ's global search.
+        excludeDirs.addAll(
+            files(
+                ".idea",
+                ".husky",
+                ".kotlin",
+                "dokka-runners/dokka-gradle-plugin/.kotlin",
```

Comment:
Q: shouldn't those be excluded by default? May be there is an issue on Idea YT (or Gradle issues)? Or should we create one?

Reply:
Yeah, it should be fixed in IntelliJ, but it hasn't been yet. I'm sure I saw a YouTrack issue for it, but I can't find it - it's impossible to search for.